packages/core/package.json

```diff
@@ -27,7 +27,7 @@
   "scripts": {
     "dev": "rollup -cw",
     "build": "rollup -c",
-    "test": "jest",
+    "test": "jest src/transforms/__tests__/resize.spec.ts",
```

Comment:
```suggestion
    "test": "jest",
```
You probably forgot to remove this before pushing 😉

Reply:
Shoot, sorry about the sloppy check-ins here!  Thanks for catching this